public/app/features/panel/state/getAllSuggestions.test.ts

```diff
@@ -101,6 +103,7 @@ scenario('Single frame with time and number field', (ctx) => {
       SuggestionName.BarGaugeLCD,
       SuggestionName.Table,
       SuggestionName.StateTimeline,
+      SuggestionName.StatusHistory,
```

Comment:
@hugohaggmark here are the updated tests that adds status history to the correct data scenarios

Reply:
Sorry I missed that 🙈 , the tests are so clean ⭐